src/file-handlers.ts

```diff
@@ -0,0 +1,22 @@
+import { flattenObject } from "flatten-anything";
```

Comment:
An operation that allows you to modify the contents of a JSON file more easily would also be useful I think:

```ts
files: {
  'package.json': fileOperations.modifyJson((pkg) => {
    delete pkg.foo.bar;

    return pkg;
  }),
},
```

Reply:
very good idea 👍